doc/api/webcrypto.md

```diff
@@ -161,7 +161,7 @@ async function generateRsaKey(modulusLength = 2048, hash = 'SHA-256') {
 ### Encryption and decryption
 
 ```js
-const crypto = require('node:crypto').webcrypto;
+const crypto = globalThis.crypto;
```

Comment:
Should we even bother with the variable declaration? I guess that clarifies that we're talking about WebCrypto and not `node:crypto`.